utils/dotenv.ts

```diff
@@ -0,0 +1,19 @@
+// https://github.com/motdotla/dotenv/issues/133#issuecomment-255298822
+
+import { join } from "path";
+import { readdirSync } from "fs";
+
+import dotenv from "dotenv";
+
+import { legacyDirname } from "./misc.js";
+const __dirname = legacyDirname(import.meta);
+
+// Traverse to parent directories until a `.env` file is found.
+// See: https://github.com/motdotla/dotenv/issues/238#issuecomment-348017037
+let currentDir = __dirname;
+while (!readdirSync(currentDir).includes(".env")) {
+  const newCurrentDir = join(currentDir, "..");
+  if (currentDir === newCurrentDir) break;
+  currentDir = newCurrentDir;
+}
```

Comment:
This is needed as we now put built files in `build` directory. We can hardcode the directory path, but we're likely to miss it if things change.